<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IVtoPID</class>
 <widget class="QMainWindow" name="IVtoPID">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>708</width>
    <height>446</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>708</width>
    <height>446</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>708</width>
    <height>446</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>IVs to PID</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
	background-color: #274050;
}

QTableView {
	border: 4px solid DodgerBlue;
}

QHeaderView::section {
	border: none;
	background-color: #274050;
	padding-top: 3px;
	padding-bottom: 3px;
	color: white;
}

QLineEdit {
	qproperty-alignment: AlignCenter;
	background-color: #1D303C;
	border:none;
	border-radius: 6px;
	color: white;
	font-family: &quot;Myriad Pro&quot;, Montserrat;
	font-weight: 500;
}

QComboBox {
	background-color: #1D303C;
	border: none;
	border-radius: 6px;
	color: white;
	padding-left: 13px;
	font-family: &quot;Myriad Pro&quot;, Montserrat;
	font-weight: 500;
	combobox-popup: 0;
}

QComboBox QListView{
	color: white;
    background-color: #1D303C;
    border:1px solid black;
}

QComboBox::item:alternate {
    background: #1D303C;
}

QComboBox::item:selected {
    border: 1px solid transparent;
    background: #003941;
}

QComboBox::indicator{
    background-color:transparent;
    selection-background-color:transparent;
    color:transparent;
    selection-color:transparent;
}

QComboBox::editable {
	background-color: #1D303C;
	border: none;
	border-radius: 6px;
	color: white;
}

QComboBox::down-arrow {
	image: url(noimg);
	border-width: 0px;
}

QComboBox::drop-down {
	background: transparent;
	width: 0px;
}

QLabel {
	color: white;
	font-family: &quot;Myriad Pro&quot;, Montserrat;
	font-weight: 500;
}

QPushButton {
	border: none;
	border-radius: 6px;
	color: white;
	background-color: DodgerBlue;
	font-family: &quot;Myriad Pro&quot;, Montserrat;
	font-weight: 500;
}

QSpinBox {
	qproperty-alignment: AlignCenter;
	background-color: #1D303C;
	border: none;
	border-radius: 6px;
	color: white;
	font-family: &quot;Myriad Pro&quot;, Montserrat;
	font-weight: 500;
}

QSpinBox::up-button {
	subcontrol-origin: border;
    subcontrol-position: top right;
}

QSpinBox::down-button {
	subcontrol-origin: border;
    subcontrol-position: bottom right;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="2" column="0" colspan="4">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>HP</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Atk</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Def</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>SpA</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>SpD</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Spe</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QSpinBox" name="spinBoxHP"/>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="spinBoxAtk"/>
      </item>
      <item row="1" column="2">
       <widget class="QSpinBox" name="spinBoxDef"/>
      </item>
      <item row="1" column="3">
       <widget class="QSpinBox" name="spinBoxSpA"/>
      </item>
      <item row="1" column="4">
       <widget class="QSpinBox" name="spinBoxSpD"/>
      </item>
      <item row="1" column="5">
       <widget class="QSpinBox" name="spinBoxSpe"/>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="labelNature">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Nature</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="labelID">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>TID</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QComboBox" name="comboBoxNatureGenerator">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>0</height>
       </size>
      </property>
      <item>
       <property name="text">
        <string>Adamant</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Bold</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Brave</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Calm</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Careful</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Hasty</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Impish</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Jolly</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Lonely</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Mild</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Modest</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Naive</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Naughty</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Quiet</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Rash</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Relaxed</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Sassy</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Timid</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Gentle</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Lax</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Bashful</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Docile</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Hardy</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Quirky</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Serious</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QPushButton" name="pushButtonFind">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>23</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Find</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="4">
     <widget class="TableView" name="tableView">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="sortingEnabled">
       <bool>false</bool>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>100</number>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>20</number>
      </attribute>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>20</number>
      </attribute>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QTextBox" name="textBoxID">
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>708</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QTextBox</class>
   <extends>QLineEdit</extends>
   <header>Forms/Controls/QTextBox.hpp</header>
  </customwidget>
  <customwidget>
   <class>TableView</class>
   <extends>QTableView</extends>
   <header>Forms/Controls/TableView.hpp</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>comboBoxNatureGenerator</tabstop>
  <tabstop>textBoxID</tabstop>
  <tabstop>spinBoxHP</tabstop>
  <tabstop>spinBoxAtk</tabstop>
  <tabstop>spinBoxDef</tabstop>
  <tabstop>spinBoxSpA</tabstop>
  <tabstop>spinBoxSpD</tabstop>
  <tabstop>spinBoxSpe</tabstop>
  <tabstop>pushButtonFind</tabstop>
  <tabstop>tableView</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
